.github/workflows/ci.yml

```diff
@@ -16,7 +16,16 @@ jobs:
       run: pip install -r tools/requirements.txt
     - name: check .rst document syntax
       run: tools/check-rst-syntax.sh
-    - name: generate and check intrinsic specs (rst)
+    - name: setup nvm
+      shell: bash -l -eo pipefail {0}
+      run: nvm install 8.10.0 && nvm use 8.10.0
+    - name: install ci dependencies
+      run: npm install
+    - name: add module to project
+      run: npm install --save-dev markdown-link-check
+    - name: install markdown-link-check
+      run: npm install -g markdown-link-check
```

Comment:
1. Please use only a single `run` step. If you cannot have all these commands in a step, export them in a separate shell scripts inside the `tools` folder and invoke such script here.
2. Which of the two `npm install ...` invocation is the one working? Remove the one that is not needed.